design/vgdp-micro-service/vgdp-micro-service.md

```diff
@@ -176,6 +176,27 @@ Below diagram shows how VGDP logs are redirected:
 
 This log redirecting mechanism is thread safe since the hook acquires the write lock before writing the log buffer, so it guarantees that in the node-agent log there is no corruptions after redirecting the log, and the redirected logs and the original node-agent logs are not projected into each other.     
 
+### Resource Control
+The CPU/memory resource of backupPod/restorePod is configurable, which means users are allowed to configure resources per volume backup/restore.  
+By default, the [Best Effort policy][5] is used, and users are allowed to change it through the ```node-agent-config``` configMap. Specifically, we add below structures to the configMap:  
+``` 
+type Configs struct {
+	// PodResources is the resource config for various types of pods launched by node-agent, i.e., data mover pods.
+	PodResources *PodResources `json:"podResources,omitempty"`
+}
+
+type PodResources struct {
+	CPURequest    string `json:"cpuRequest,omitempty"`
```

Comment:
Should we also allow users to set requests and limits for `emphemeral-storage` as well ?

Reply:
Do you mean the root file system of the pod? We already have the backup repo config for users to set the cacheLimit which consumes the root file system space. Isn't it enough?

And for the current change in 1.15, we just need to fix the gap comparing to the previous releases, for any new functionalities, we can add post 1.15.

Reply:
I meant this https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage

Reply:
Yes, that is what I meant root file system of the pod. We don't need to let users config it, instead, we let users to config the `cacheLimit` which consumes the most proportion of the root file system space.

Reply:
@shubham-pampattiwar Does the above comment answer your question? Or do you have more comments? Once you complete the review, please help to approve it.

Reply:
looks good, Ack!